src/components/ChattingScreen.js

```diff
@@ -1,31 +1,103 @@
-import React from 'react';
-import styled from 'styled-components';
+import React, { useState, useEffect } from "react";
+import styled from "styled-components";
 
-import Header from './Header';
-import MessageSender from './MessageSender';
+import Header from "./Header";
+import MessageSender from "./MessageSender";
 
 export default function ChattingScreen() {
-	const EUNKO = 'https://img.techpowerup.org/200908/eun.png';
-	const COOL = 'https://img.techpowerup.org/200908/NjRiY2JjOGU5YzQz.png';
-
-	const MSGLIST = [
-		{ user: true, content: '안녕하세요 12기 프론트엔드 개발자분들' },
-		{ user: true, content: '저희의 대화를 마음껏 조작해보세요 💌' },
-		{ user: true, content: '상단에 프로필을 눌러서 발신자 변경하면 됩니당~' },
-		{ user: false, content: '안녕 은아' },
-		{ user: false, content: '뭐해 ?' },
-		{ user: true, content: '시원아 넌 최고의 팀장이야' },
-		{ user: false, content: '나도 아니까  ' },
-		{ user: false, content: '그만 말해줘도 돼' },
-		{ user: true, content: 'ㅠㅠ' },
-	];
-
-	return (
-		<Wrapper>
-			<Header></Header>
-			<MessageSender></MessageSender>
-		</Wrapper>
-	);
+  const MSGLIST = [
+    { user: true, content: "안녕하세요 12기 프론트엔드 개발자분들" },
+    { user: true, content: "저희의 대화를 마음껏 조작해보세요 💌" },
+    { user: true, content: "상단에 프로필을 눌러서 발신자 변경하면 됩니당~" },
+    { user: false, content: "안녕 은아" },
+    { user: false, content: "뭐해 ?" },
+    { user: true, content: "시원아 넌 최고의 팀장이야" },
+    { user: false, content: "나도 아니까  " },
+    { user: false, content: "그만 말해줘도 돼" },
+    { user: true, content: "ㅠㅠ" },
+  ];
+  const [InputText, setInputText] = useState("");
+  const [MessageList, setMessageList] = useState(MSGLIST);
+  const [User, setUser] = useState(true);
+
+  useEffect(() => {
+    window.scrollBy({
+      botton: 100,
+      behavior: "smooth",
+    });
+  }, [MessageList]);
+
+  const handleInput = (event) => {
+    event.preventDefault();
+
+    setInputText(event.target.value);
+  };
+
+  const handleSubmit = (event) => {
+    //prevent refresh right after submit
+    event.preventDefault();
+
+    //handle exception when user typed nothing
+    if (InputText === "") {
+      alert("내용을 입력하세요");
+      return;
+    }
+    //concat to previous MessageList
+    const nextMessageList = MessageList.concat({
+      user: User,
+      content: InputText,
+    });
+    //update MessageList
+    setMessageList(nextMessageList);
+    setInputText("");
+  };
+
+  return (
+    <>
+      {/* child component updates state in parent component   */}
+      <Header user={User} onClick={() => setUser(!User)}></Header>
+
+      <Chat>
+        <MessageSender {...MessageList}></MessageSender>
```

Comment:
이렇게하시면 messageSender에서 처리하고 있는 로직이 필요없어집니다 :)
그리고 messageSender를 inputBox해주신걸로 만들어주시면 될 것 같아요
styled-component에도 props를 전달해줄 수 있어요..!
아래 스타일도 리뷰남겨드렸으니 함께 확인해주세요 💫

```suggestion
        {MessageList.map((message, index) => {
					return (
						<MessageRow key={index} sending={!message.user}>
							<Img src={message.user ? EUNKO : COOL} />
							<Message>{message.content}</Message>{' '}
						</MessageRow>
					);
				})}
```